test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -328,3 +334,24 @@ def verify_collectibles_exist(self, account_name: str):
             collectionsRepeater.itemAt(0).expanded = True
         collectiblesRepeater = get_obj(CollectiblesView.COLLECTIBLES_REPEATER.value)
         verify(collectiblesRepeater.count > 0, "Collectibles not retrieved for the account")
+                    
+    def verify_transactions_exist(self):
+        tabbar = get_obj(MainWalletScreen.RIGHT_SIDE_TABBAR.value)
+        click_obj(tabbar.itemAt(WalletTabBar.ACTIVITY_TAB.value))
+
+        transactionListView = get_obj(TransactionsView.TRANSACTIONS_LISTVIEW.value)
+        
+        squish.waitFor("transactionListView.count > 0", 60*1000)
+        verify(transactionListView.count > 0, "Transactions not retrieved for the account")
+        
+        if(transactionListView.count > 1):
```

Comment:
do u need this if check? as the line before says:
```verify(transactionListView.count > 0, "Transactions not retrieved for the account")```

Reply:
yes, I think can make 
`verify(transactionListView.count > 0, "Transactions not retrieved for the account")` to 
verify(transactionListView.count > 1, "Transactions not retrieved for the account") instead